src/js/index.js

```diff
@@ -9,92 +11,437 @@ const deepFreeze = obj => {
+      },
+      'orange!': '#FF8300',
+      yellow: {
+        dark: '#D78F00',
+        light: '#FFEB59',
+      },
+      'yellow!': '#FEC901',
+      'graph-0': 'orange',
+      'graph-1': 'blue',
+      'graph-2': 'purple',
+      focus: 'green',
+      placeholder: 'text-weak',
+    },
     font: {
       family: "'Metric', Arial, sans-serif",
       face: `
         @font-face {
           font-family: "Metric";
           src: url("https://hpefonts.s3.amazonaws.com/web/MetricHPE-Web-Regular.woff") format('woff');
         }
-
         @font-face {
           font-family: "Metric";
           src: url("https://hpefonts.s3.amazonaws.com/web/MetricHPE-Web-Bold.woff") format('woff');
           font-weight: 700;
         }
-
         @font-face {
           font-family: "Metric";
           src: url("https://hpefonts.s3.amazonaws.com/web/MetricHPE-Web-Semibold.woff") format('woff');
           font-weight: 600;
         }
-
         @font-face {
           font-family: "Metric";
           src: url("https://hpefonts.s3.amazonaws.com/web/MetricHPE-Web-Light.woff") format('woff');
           font-weight: 100;
-        }
-      `,
+        }`,
+    },
+    active: {
+      background: 'active-background',
+      color: 'active-text',
+    },
+    drop: {
+      background: 'background-front',
+      border: {
+        radius: '8px',
+      },
+      shadowSize: 'medium',
+    },
+    hover: {
+      background: 'active-background',
+      color: 'active-text',
+    },
+    selected: {
+      background: 'selected-background',
+      color: 'selected-text',
+    },
+  },
+  // Missing from the Designer
+  accordion: {
+    border: {
+      color: 'text',
+    },
+    icons: {
+      color: 'text',
     },
   },
+  // Missing from the Designer
   anchor: {
-    textDecoration: 'underline',
-    color: {
-      dark: '#FFFFFF',
-      light: '#000000',
+    color: 'text',
+    fontWeight: 700,
+    textDecoration: 'none',
+    hover: {
+      textDecoration: 'none',
     },
   },
+  // Radius + font is missing from the designer
   button: {
     border: {
-      radius: '0px',
+      radius: '6px',
     },
-    colors: {
-      accent: '#ff8d6d',
-      secondary: 'rgba(51,51,51,0.6)',
+    extend: css`
+      ${props => !props.plain && 'font-weight: bold;'}
+    `,
+  },
+  calendar: {
+    small: {
+      fontSize: '11.6px',
+      lineHeight: 1.375,
+      daySize: '27.43px',
+    },
+    medium: {
+      fontSize: '18px',
+      lineHeight: 1.45,
+      daySize: '54.86px',
+    },
+  },
+  // Designer defaults to base.js, the following is extra
+  checkBox: {
+    gap: 'medium',
+    color: 'text-strong',
+  },
+  formField: {
+    border: {
+      side: 'all',
     },
-    extend: 'letter-spacing: 0.04167em;',
   },
-  clock: {
-    analog: {
-      second: {
-        color: {
-          dark: '#01a982',
-          light: '#01a982',
+  heading: {
+    level: {
+      '1': {
+        small: {
+          size: '43.6px',
```

Comment:
So, I would think all of these sizes would be the same as base.js, since grommet also uses 24px base spacing?

Reply:
Partly accurate, the `baseSpacing` is the same, but these values are caculated according to `fontSizing`, and the fontSizing of the HPE theme is different than the base.js.
e.g.
![Screen Shot 2020-02-04 at 4 44 05 PM](https://user-images.githubusercontent.com/6320236/73797639-d93ecb80-476d-11ea-9119-246e51a09cec.png)

Reply:
Ah, OK.